src/App.scss

```diff
@@ -2,36 +2,123 @@ iframe {
   display: none;
 }
 
+.title {
+  font-size: 50px;
+}
+
+.containerForApp {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App {
+  border: 2px black solid;
+  display: flex;
+  flex-direction: column;
+  max-width: 75%;
+  align-items: center;
+}
+
+.formUsers {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  gap: 20px;
+
+  .field {
+    font-family: "Franklin Gothic Medium", Arial, sans-serif;
+    font-size: 30px;
+  
+    &-form {
+      display: block;
+      width: 100%;
+      background-color: aqua;
+      padding: 8px 16px;
+      line-height: 25px;
+      font-size: 14px;
+      font-weight: 500;
+      font-family: inherit;
+      border-radius: 6px;
+      -webkit-appearance: none;
```

Comment:
Avoid using browser-specific hacks like '-webkit-appearance'. Consider using a library like Autoprefixer to handle browser compatibility issues.